### ISSUE TITLE
Feature: Display path hint when multiple tabs have the same folder name

### ISSUE THREAD

**Issue by Bibipp45** [files-community/Files#14175]
### What feature or improvement do you think would benefit Files?

Would it be possible to display the full path for each tab as original on the file explorer and not just the name of the open folder please.

### Requirements

- Display part of path when multiple tabs have the same name
- Use the following format "folder name" "drive letter" "first letter of parent folder"

### Files Version

3.0.8

### Windows Version

10.0.22631.2715

### Comments

![image](https://github.com/files-community/Files/assets/39923744/7905e2e4-98df-476f-ab6b-e493a347af67)

**Comment by yaira2** [comment]
Thank you for the feedback, we'll consider this suggestion but in case you weren't aware, you can view the full path by hovering over the tab.

> original on the file explorer and not just the name of the open folder please.

Is there a way to enable this in File Explorer?

**Comment by Bibipp45** [comment]
Yes I know that the path is visible when hovering but it is not practical
when moving a file or folder.
On the original explorer the option to enable full path is here: "...
/Options /Display/Show full path in title bar"

**Comment by yaira2** [comment]
I suppose we can display a hint without displaying the full path (similar to VS Code), would this help for your use case?
![image](https://github.com/files-community/Files/assets/39923744/e389a7c6-ee82-4c64-b243-804d792c96b1)

**Comment by Bibipp45** [comment]
Yes it can be quite good, to try

**Comment by natebizzle** [comment]
I'd like to expand on this request and provide a good example of the implementation from XYplorer where you can have multiple tabs within a dual pane and the path bar is added to the bottom of the tab row -

![Tabs within dual pane view_Path](https://github.com/files-community/Files/assets/99941017/cc53d564-3273-4cab-b98a-967ce2872212)

Each directory within the path bar is also clickable so you can jump directly to any parent folder all the way up to the parent. Allowing tabs within the dual panes also provides much more flexibility for combinations than switching between the current tab implementation with a single directory for each pane.

**Comment by Bibipp45** [comment]
Nice, good job

Le sam. 13 janv. 2024, 09:06, natebizzle ***@***.***> a
écrit :

**Comment by yair100** [comment]
This feature will be included in the next update.
